D/DomainSets/Compat.toml

```diff
@@ -1,4 +1,9 @@
 ["0.0"]
 IntervalSets = "0.3"
 StaticArrays = "0.8.3-0.10"
+
+["0.0.1"]
 julia = "0.7-1.3"
+
+["0.0.2"]
+julia = "0.7-1.1"
```

Comment:
This is a bug https://github.com/JuliaComputing/Registrator.jl/issues/103 so please retrigger the bot when the linked issue is fixed.